infectiousdiseases/patient_lists.py

```diff
@@ -0,0 +1,62 @@
+from opal.core.patient_lists import PatientList, TaggedPatientList
+from elcid import models
+
+
+class Virology(TaggedPatientList):
+    tag = "virology"
+    schema = [
+        models.Demographics,
+        models.Location,
+        models.Diagnosis,
+        models.Antimicrobial,
+        models.MicrobiologyTest,
+        models.MicrobiologyInput,
+        models.GeneralNote
+    ]
+
+
+class Microbiology(TaggedPatientList):
+    tag = "microbiology"
```

Comment:
I think this is exactly the kind of thing that shouldn't have a list.
There is no Microbiology list, but there is a microbiology service that has child lists.
And there is a MicroOrtho list tagged micro_ortho